services/frontend/src/components/SignIn.vue

```diff
@@ -37,8 +41,37 @@
 </template>
 
 <script>
+import router from '../router'
+
 export default {
-  name: 'Login'
+  name: 'Login',
+  data() {
+    return {
+      username: null,
+      password: null
+    }
+  },
+  methods: {
+    checkLogin() {
+      // TODO: this is a mock
```

Comment:
Can you add a comment like this to other places with mocks ?

Reply:
i think i did, but will check to be sure

Reply:
ijn `verifyPatient` theres no comment